stanford/roles/cluster/defaults/main.yml

```diff
@@ -21,11 +21,11 @@ CLUSTER_SECURITY_GROUPS:
 
 CLUSTER_NETWORK: private
 CLUSTER_SUBNETS:
-  public:
+  build:
```

Comment:
`git` is just confused on this stanza; we're moving `public` into the `deployment` role, per discussion.